{
  "schema_version": "1.6.3",
  "id": "USN-4172-1",
  "summary": "file vulnerability",
  "details": "It was discovered that file incorrectly handled certain malformed files. An\nattacker could use this issue to cause a denial of service, or possibly\nexecute arbitrary code.\n",
  "aliases": [],
  "related": [
    "CVE-2019-18218",
    "UBUNTU-CVE-2019-18218"
  ],
  "published": "2019-10-30T13:49:35.863465Z",
  "modified": "2019-10-30T13:49:35.863465Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:16.04:LTS",
        "name": "file",
        "purl": "pkg:deb/ubuntu/file@1:5.25-2ubuntu1.3?arch=source&distro=xenial"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1:5.25-2ubuntu1.3"
            }
          ]
        }
      ],
      "versions": [
        "1:5.22+15-2ubuntu1",
        "1:5.25-2ubuntu1",
        "1:5.25-2ubuntu1.1",
        "1:5.25-2ubuntu1.2"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "binary_name": "file",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "file-dbg",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "file-dbgsym",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "libmagic-dev",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "libmagic-dev-dbgsym",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "libmagic1",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "libmagic1-dbgsym",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "python-magic",
            "binary_version": "1:5.25-2ubuntu1.3"
          },
          {
            "binary_name": "python3-magic",
            "binary_version": "1:5.25-2ubuntu1.3"
          }
        ],
        "availability": "No subscription required"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:18.04:LTS",
        "name": "file",
        "purl": "pkg:deb/ubuntu/file@1:5.32-2ubuntu0.3?arch=source&distro=bionic"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1:5.32-2ubuntu0.3"
            }
          ]
        }
      ],
      "versions": [
        "1:5.32-1",
        "1:5.32-2",
        "1:5.32-2ubuntu0.1",
        "1:5.32-2ubuntu0.2"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "binary_name": "file",
            "binary_version": "1:5.32-2ubuntu0.3"
          },
          {
            "binary_name": "file-dbgsym",
            "binary_version": "1:5.32-2ubuntu0.3"
          },
          {
            "binary_name": "libmagic-dev",
            "binary_version": "1:5.32-2ubuntu0.3"
          },
          {
            "binary_name": "libmagic-mgc",
            "binary_version": "1:5.32-2ubuntu0.3"
          },
          {
            "binary_name": "libmagic1",
            "binary_version": "1:5.32-2ubuntu0.3"
          },
          {
            "binary_name": "libmagic1-dbgsym",
            "binary_version": "1:5.32-2ubuntu0.3"
          }
        ],
        "availability": "No subscription required"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://ubuntu.com/security/notices/USN-4172-1"
    },
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2019-18218"
    }
  ]
}